lib/matplotlib/axes/_axes.py

```diff
@@ -1868,25 +1868,47 @@ def step(self, x, y, *args, **kwargs):
 
         return self.plot(x, y, *args, **kwargs)
 
-    @_preprocess_data(replace_names=["left", "height", "width", "bottom",
+    @_preprocess_data(replace_names=["x", "left",
+                                     "height", "width",
+                                     "y", "bottom",
                                      "color", "edgecolor", "linewidth",
                                      "tick_label", "xerr", "yerr",
                                      "ecolor"],
-                      label_namer=None)
+                      label_namer=None,
+                      positional_parameter_names=['x',
+                                                  'height', 'width',
+                                                  'y'])
     @docstring.dedent_interpd
-    def bar(self, left, height, width=0.8, bottom=None, **kwargs):
-        """
-        Make a bar plot.
+    def bar(self, *args, **kwargs):
+        """Make a bar plot.
+
+        Call signatures::
+
+           bar(x, height, *, align='center', **kwargs)
+           bar(x, height, width, *, align='center', **kwargs)
+           bar(x, height, width, bottom, *, align='center', **kwargs)
 
         Make a bar plot with rectangles bounded by:
 
-          `left`, `left` + `width`, `bottom`, `bottom` + `height`
+          `x` - width/2, `x` + `width`/2, `bottom`, `bottom` + `height`
```

Comment:
These should be changed to use double backquotes around each complete expression.